distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -62,11 +62,10 @@ type File
 
              example_new = File.new Examples.csv_path
     new : (Text | File) -> File
-    new path =
+    new path:(Text|File) =
```

Comment:
We should probably discuss about the best practices.

For me, the preferred syntax in cases like this would be
```suggestion
    new (path : Text|File) =
```

IMO it is more readable, it is closer to languages like Scala. But it is just a personal preference, I'm open to discussion. But I think ideally we should agree on one convention and try to use it consistently as a team. 

I think the `(x : A)` one is used a bit more commonly throughout the codebase, that's why I'm nitpicking here.